packages/daemon/__tests__/integration/wait-for-db-up.ts

```diff
@@ -0,0 +1,42 @@
+import mysql from 'mysql2/promise';
+import {
+  DB_USER,
+  DB_PASS,
+  DB_PORT,
+  DB_ENDPOINT,
+} from './config';
+
+const attemptConnection = async (maxAttempts: number, interval: number): Promise<void> => {
+  let attempts = 0;
+
+  while (attempts < maxAttempts) {
+    try {
+      const conn = await mysql.createConnection({
+        host: DB_ENDPOINT,
+        user: DB_USER,
+        password: DB_PASS,
+        port: DB_PORT,
+      });
+      console.log('Successfully connected to the database!');
```

Comment:
suggestion: Should we make a `SELECT 1` query to test the connection?

Reply:
Yes! Done in 5810e307c9b6ddb68fafd25c08133644396847f0